src/components/app/data/services/subsidies/browseAndRequest.js

```diff
@@ -16,11 +17,10 @@ export async function fetchBrowseAndRequestConfiguration(enterpriseUUID) {
     const response = await getAuthenticatedHttpClient().get(url);
     return camelCaseObject(response.data);
   } catch (error) {
-    const errorResponseStatusCode = getErrorResponseStatusCode(error);
-    if (errorResponseStatusCode === 404) {
-      return null;
+    if (getErrorResponseStatusCode(error) !== 404) {
+      logError();
```

Comment:
[nit] message isn't passed into the log error

Reply:
D'oh, great catch!